shell/platform/darwin/macos/framework/Headers/FlutterViewController.h

```diff
@@ -84,13 +84,26 @@ FLUTTER_DARWIN_EXPORT
 @property(nonatomic) FlutterMouseTrackingMode mouseTrackingMode;
 
 /**
- * Initializes a controller that will run the given project.
+ * Initializes this `FlutterViewController` for the implicit view of a new
+ * `FlutterEngine`.
```

Comment:
I prefer the original sentence here:

> Initializes a controller that will run the given project.

I don't think we're ready yet to expose the implicit view to the user. This isn't something folks have needed to know about today. Could we punt this "implicit view" messaging until we're ready to migrate folks off the implicit view?